Decompiler/Decompiler.cs

```diff
@@ -889,6 +889,7 @@ private void DumpVPK(string parentPath, Package package, string type, Dictionary
                     catch (Exception e)
                     {
                         LogException(e, filePath, package.FileName);
+                        contentFile?.Dispose();
```

Comment:
This seems suspicious because there's using contentFile below, should contentFile be reset to null, outputfile set to null, or check that contentFile isn't null below?

Reply:
Oh it still goes through... lol. I assume it was meant to output the file "as is" in case contentFile extraction fails. In that case a null check should do.

Reply:
Generally for disposables it should be wrapped in using for the all the code that accesses the variable.

That or, you can manually write out try/finally with a dispose call. Right now this code still looks like it could miss a dispose if something throws in the middle.

Reply:
Should be fine now. Realistically, if those things throw the program would quit anyway though, I think.